src/metabase/public_settings.clj

```diff
@@ -32,12 +32,14 @@
   (when-let [varr (resolve 'metabase-enterprise.sso.integrations.sso-settings/other-sso-configured?)]
     (varr)))
 
-(defn- sso-configured?
-  "Any SSO provider is configured"
-  []
-  (or (google-auth-configured?)
-      (ldap-configured?)
-      (ee-sso-configured?)))
+(defsetting sso-configured?
+  (deferred-tru "Are we using any SSO integration?")
+  :type       :boolean
+  :visibility :public
+  :setter     :none
+  :getter (fn [] (or (google-auth-configured?)
+                     (ldap-configured?)
+                     (ee-sso-configured?))))
```

Comment:
The whole body is taken from `ldap-configured?`, and I only changed`:getter` 
https://github.com/metabase/metabase/blob/9dd96b3c68007ead23d2af055fe94e97234fef44/src/metabase/integrations/ldap.clj#L97-L104